conf/application.conf

```diff
@@ -143,6 +143,7 @@ nino-logging.enabled = false
 
 stride {
   base64-encoded-roles        = [ "aHRzIGhlbHBkZXNrIGFkdmlzb3I=" ]
+  base64-encoded-secure-roles = [ "aHRzIGhlbHBkZXNrIGFkdmlzb3Igc2VjdXJl" ]
```

Comment:
is the below config still required ? 

```
manual-account-creation {
   enabled = true
 }
```

Reply:
so, can we delete it now, if its no longer required ?

Reply:
I thought it was deleted..let me take a look

Reply:
nope, you're right - it's still there. Just deleted and pushed.